scripts/geolocation.js

```diff
@@ -0,0 +1,25 @@
+const { paintData } = require('./paintData');
+const { getResponse } = require('./responseApi');
+
+function getGeo() {
+  navigator.geolocation.getCurrentPosition((posityion) => {
+    const {
+      coords: { latitude, longitude },
+    } = posityion;
+    getResponse(`${latitude},${longitude}`).then((response) => {
+      paintData(response);
+    });
+  });
+}
+
+function paintGeolocationData() {
+  document
+    .querySelector('.header-geolocation')
+    .addEventListener('click', (e) => {
+      if (e.target.closest('.header-geolocation')) {
+        getGeo();
+      }
+    });
+}
```

Comment:
```suggestion
const getGeo = () => {
  navigator.geolocation.getCurrentPosition((posityion) => {
    const {
      coords: { latitude, longitude },
    } = posityion;
    getResponse(`${latitude},${longitude}`).then((response) => {
      paintData(response);
    });
  });
};

const paintGeolocationData = () => {
  document
    .querySelector('.header-geolocation')
    .addEventListener('click', (e) => {
      if (e.target.closest('.header-geolocation')) {
        getGeo();
      }
    });
};
```
Не бойся использовать стрелочные функции если не работаешь с контекстом